framework/data/Sort.php

```diff
@@ -30,8 +30,8 @@
  *         'attributes' => [
  *             'age',
  *             'name' => [
- *                 'asc' => ['first_name' => SORT_ASC, 'last_name' => SORT_ASC],
- *                 'desc' => ['first_name' => SORT_DESC, 'last_name' => SORT_DESC],
+ *                 'asc' => ['first_name' => SORT_ASC, '[[last_name]] ASC'],
+ *                 'desc' => ['first_name' => SORT_DESC, '[[last_name]] DESC'],
```

Comment:
Also it is better to mention PostgreSQL specific example with `NULLS FIRST` involved to make it more clear when it may be needed.